.github/workflows/mingw.yml

```diff
@@ -0,0 +1,293 @@
+          MSYSTEM= ./python.exe "$SMOKETESTS"
+
+      - name: Compress
+        if: always()
+        shell: msys2 {0}
+        run: |
+          cd _build
+          tar -zcf python.tar.gz python_pkgdir/
+
+      - name: Upload
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: build-${{ matrix.msystem }}
+          path: _build/python.tar.gz
+
+  cross-gcc-x86_64:
+    runs-on: ubuntu-latest
+    container:
+      image: archlinux:base-devel
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install deps
+        run: |
+          pacman --noconfirm -Suuy
+          pacman --needed --noconfirm -S mingw-w64-gcc autoconf-archive autoconf automake zip
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Check Python Version
+        run: |
+          which python
+          python --version
+
+      - name: Build
+        run: |
+          autoreconf -vfi
+
+          mkdir _build && cd _build
+
+          ../configure \
+            --host=x86_64-w64-mingw32 \
+            --build=x86_64-pc-linux-gnu \
+            --enable-shared \
+            --with-system-expat \
+            --with-system-ffi \
+            --with-system-libmpdec \
+            --without-ensurepip \
+            --enable-loadable-sqlite-extensions
+
+          make -j8
+
+          make install DESTDIR="$(pwd)/install"
+
+      - name: 'Zip files'
+        run: |
+          zip -r install.zip _build/install
+
+      - name: Upload
+        uses: actions/upload-artifact@v3
+        with:
+          name: build-cross-gcc-x86_64
+          path: install.zip
+
+  cross-gcc-x86_64-test:
+    needs: [cross-gcc-x86_64]
+    runs-on: windows-latest
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: build-cross-gcc-x86_64
+
+      - name: 'Run tests'
+        run: |
+          7z x install.zip
+          ./_build/install/usr/local/bin/python3.exe -c "import sysconfig, pprint; pprint.pprint(sysconfig.get_config_vars())"
+
+
+  cross-llvm-mingw:
+    runs-on: ubuntu-latest
+    container:
+      image: mstorsjo/llvm-mingw:latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: ['x86_64', 'i686', 'aarch64', 'armv7']
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install deps
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          apt-get update -qq
+          apt-get install -qqy software-properties-common
+          add-apt-repository --yes ppa:deadsnakes/ppa
+          apt-get update -qq
+          apt-get install -qqy autoconf-archive python3.11-dev python3.11
+
+      - name: Build
+        run: |
+          autoreconf -vfi
+
+          mkdir _build && cd _build
+
+          export CC=${{ matrix.arch }}-w64-mingw32-clang
+          export CXX=${CC}++
+          ../configure \
+            --host=${{ matrix.arch }}-w64-mingw32 \
+            --build=x86_64-pc-linux-gnu \
```

Comment:
```suggestion
            --build=x86_64-pc-linux-gnu \
            --with-build-python="$(which python)" \
```
?

Reply:
it's python3 with Ubuntu, and that's 3.8, so I went with the versioned name.

Reply:
cool.  The docs seemed to suggest it wanted a path, so I wasn't sure if it was going to work without an absolute path,